content/en/xtc/xtc-release-notes/v54.md

```diff
@@ -0,0 +1,34 @@
+---
+title: XTC v54
+linkTitle: XTC v54
+
+weight: 162
+type: docs
+
+date: 2022-08-22
+
+description: 2022-08-23
+
+sitemap:
+  changefreq: never
+  priority: 0.1
+---
+
+## Platform
+
+* Minor UI fixes and improvements.
+
+
+## Load Testing
+
+* The load test table now lists the duration of finished load tests in a new sortable column.
+* Before a load test can be started, the underlying load test suite needs to be built which may take several minutes to complete. Most of that time can be attributed to the download of dependencies (XLT and other required libraries) from Maven Central. XTC now caches the downloaded dependencies so subsequent builds should run much faster. If you need to discard the cache, you can do so in the new section _Settings_ > _Repository_ > _Build Dependency Cache_ of your load testing project.
+* Custom reports will be created from the latest available result. When determining the latest result, XTC now skips pending or failed results as these typically don't have result data attached (yet).
```

Comment:
```suggestion
* Custom reports will be created from the latest available result in case none was explicitly chosen. When determining the latest result, XTC now skips pending or failed results as these typically don't have result data attached (yet).
```

Reply:
Done.